src/index.ts

```diff
@@ -131,7 +130,7 @@ export class EppoJSClient extends EppoClient {
  * @param config - client configuration
  * @public
  */
-export async function init(config: IClientConfig): Promise<IEppoClient> {
+export async function init(config: IClientConfig): Promise<EppoJSClient> {
```

Comment:
Might have made a mess of things:

* `IEppoClient` (commons) has the public method definitions - `get*Assignment`
* `EppoClient` - (commons) `setLogger`, `disableAssignmentCache`, etc
* `EppoJSClient` - (this repo) - local configuration

To me it makes some sense to return the most locally relevant client `EppoJSClient`, here.

Reply:
The `EppoJSClient` changes make sense to me too. My main concern was that it would mean that clients would need to update types in their code after updating their SDK version, which is undesirable as far as making updates as low friction as possible

Are there any disadvantages to putting `disableAssignmentCache` in the common library?

Reply:
Agree with not making a breaking change here.

I do like the the idea of letting them deviate from our defaults should they want to with config options though (even though, like Eric, I don't immediately see a reason why they would other than extreme optimizations).